web/src/features/Dashboard/components/DashboardLicenseCard.tsx

```diff
@@ -291,7 +291,7 @@ const DashboardLicenseCard = (props: Props) => {
                   {appLicense?.assignee}
                 </p>
                 {appLicense?.channelName && (
-                  <span className="channelTag flex-auto alignItems--center u-fontWeight--medium u-marginLeft--10">
+                  <span className="channelTag alignItems--center u-fontWeight--medium u-marginLeft--10">
```

Comment:
removing `flex-auto` so channelTag will wrap but will allow customer name to have more width before we start wrapping.

Reply:
let me know if you think there's an alternative approach